From FEDORA_PATCHES Mon Sep 17 00:00:00 2001
From: Keith Seitz <keiths@redhat.com>
Date: Fri, 11 Jan 2019 17:02:15 -0500
Subject: gdb-rhbz1187581-power8-regs-not-in-8.2-08of15.patch

;; [PowerPC] Fix indentation in arch/ppc-linux-common.c
;; Pedro Franco de Carvalho, RH BZ 1187581

[PowerPC] Fix indentation in arch/ppc-linux-common.c

This patch parenthesizes the tdesc selection expressions in
arch/ppc-linux-common.c so that they can be tab-indented.

gdb/ChangeLog:
2018-10-26  Pedro Franco de Carvalho  <pedromfc@linux.ibm.com>

    * arch/ppc-linux-common.c (ppc_linux_match_description):
    Parenthesize tdesc assignements and indent them properly.

diff --git a/gdb/arch/ppc-linux-common.c b/gdb/arch/ppc-linux-common.c
--- a/gdb/arch/ppc-linux-common.c
+++ b/gdb/arch/ppc-linux-common.c
@@ -53,14 +53,14 @@ ppc_linux_match_description (struct ppc_linux_features features)
       if (features.cell)
 	tdesc = tdesc_powerpc_cell64l;
       else if (features.vsx)
-	tdesc = features.isa205
-	  ? tdesc_powerpc_isa205_vsx64l : tdesc_powerpc_vsx64l;
+	tdesc = (features.isa205? tdesc_powerpc_isa205_vsx64l
+		 : tdesc_powerpc_vsx64l);
       else if (features.altivec)
-	tdesc = features.isa205
-	  ? tdesc_powerpc_isa205_altivec64l : tdesc_powerpc_altivec64l;
+	tdesc = (features.isa205? tdesc_powerpc_isa205_altivec64l
+		 : tdesc_powerpc_altivec64l);
       else
-	tdesc = features.isa205?
-	  tdesc_powerpc_isa205_64l : tdesc_powerpc_64l;
+	tdesc = (features.isa205? tdesc_powerpc_isa205_64l
+		 : tdesc_powerpc_64l);
     }
   else
     {
@@ -69,14 +69,14 @@ ppc_linux_match_description (struct ppc_linux_features features)
       if (features.cell)
 	tdesc = tdesc_powerpc_cell32l;
       else if (features.vsx)
-	tdesc = features.isa205
-	  ? tdesc_powerpc_isa205_vsx32l : tdesc_powerpc_vsx32l;
+	tdesc = (features.isa205? tdesc_powerpc_isa205_vsx32l
+		 : tdesc_powerpc_vsx32l);
       else if (features.altivec)
-	tdesc = features.isa205
-	  ? tdesc_powerpc_isa205_altivec32l : tdesc_powerpc_altivec32l;
+	tdesc = (features.isa205? tdesc_powerpc_isa205_altivec32l
+		 : tdesc_powerpc_altivec32l);
       else
-	tdesc = features.isa205
-	  ? tdesc_powerpc_isa205_32l : tdesc_powerpc_32l;
+	tdesc = (features.isa205? tdesc_powerpc_isa205_32l
+		 : tdesc_powerpc_32l);
     }
 
   gdb_assert (tdesc != NULL);
